wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -37,21 +38,20 @@ Overall rules and guidelines apply to every kind of osu!catch difficulty.
 
 #### Rules
 
-- **Your map must theoretically be possible to SS.** This means it must be possible to catch absolutely all fruits, including droplets.
-- **Edge dashes must not be used in direct conjunction with hyperjumps.** This is because such patterns require especially precise movement and force an unreasonable restriction on accuracy required to catch them.
-- **Each map must use at least two different combo colours which must not blend in with the map's background/storyboard/video.** This is so hit objects are always visible to the player.
+- **Your map must theoretically be possible to SS.** This means it must be possible to catch absolutely all fruits, drops and droplets.
+- **Each map must use at least two different combo colours which must not blend in with the map's background/storyboard/video.** This is so objects are always visible to the player.
```

Comment:
I think the wording here changes the meaning of the rule unintentionally. what I assume RC means to say is
- must use at least two different combo colors
- combo colors must not blend with bg/sb/video

but what it really says with this wording is that you can have 2 combos that follow the rules and then a third color that blends in with SB/bg/video. to fix i would just split into 2 different rules

Reply:
changing to match standard's rule: Each beatmap must use at least two different custom combo colours unless the default skin is forced. The combo colours must not blend with the beatmap's background/storyboard/video in any case. This is so hit objects are always visible to the player and custom skin's combo colours do not blend with the background accidentally.